### DISCUSSION TITLE
你好站长，账号不知道为什么被封了

### DISCUSSION THREAD

**Discussion by departever** [BurnHair/docs#119] [技术支持]
报错如下{
  "error": {
    "message": "用户1892已被封禁：重复注册 (request id: 20240624073726820163701PzUOV4T8)",
    "type": "one_api_error"
  }
}。但是我一直都是正常使用的问问算法还有修改代码意见的，也没有违规对话什么的。能麻烦看一下是什么问题嘛？是因为每天摇骰子白嫖太多了还是怎么了。站长能帮忙解决一下吗，阿里嘎多！

**Comment by BennyThink** [comment]
原因是重复注册。你确定自己没有多次注册吗？

**Reply by departever** [reply]
确定没有呀，不过平常有时候next chat有时候连不上我是挂着梯子问的，是不是这个原因？我一直都是用github登录的。

**Reply by qshhb** [reply]
{
  "error": {
    "message": "用户3413已被封禁：重复注册 (request id: 20240625075340170341021FCk5Wdjs)",
    "type": "one_api_error"
  }
  你好站长，我也出现了一样的问题，一直用的github登录的，不过不知道是不是因为推荐给了两个同事使用，触发重复IP了，能麻烦站长帮忙解决一下吗，谢谢！

**Reply by BennyThink** [reply]
可能是，再试试看吧。

**Reply by qshhb** [reply]
好的可以了，谢谢站长

**Reply by departever** [reply]
谢谢站长！麻烦你了
